### PR TITLE
make final changes from product owner code review

### PR DESCRIPTION
**Pull request by HiredMussel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
+						<div class=hoverContainer>
 							<div class=textBox>
 								<h2>PROJECT TITLE</h2>
 								<p>A short description of the project would appear in this section</p>
 							</div>
-							<div class=buttonContainer>
-								<a href="#">HAVE A LOOK</a>
-							</div>
+							<a href="#">
+								<div class=buttonContainer>
```
Comment (ashleycoles) [inline index.html]: Watch out for missing quotes around classes

File: index.html [hunk 2]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
 				<h1>My Work</h1>
 				<!-- 	This section will be a flexbox containing thumbnails of completed projects. Mousing over the thumbnail will bring
 						up a short description of the project, and clicking it will go to that page -->
-					<div class=portfolioFlex>
-						<!-- This div is the actual portfolio item, and can be replicated if desired -->
-						<div class=projContainer>
-							<img src="https://via.placeholder.com/240x352" alt="Placeholder Image">
+				<div class=portfolioFlex>
+					<!-- This div is the actual portfolio item, and can be replicated if desired -->
+					<div class=projContainer>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html [hunk 3]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
-			<section id="portfolioContent">
+			<section id="portfolio">
 				<h1>My Work</h1>
 				<!-- 	This section will be a flexbox containing thumbnails of completed projects. Mousing over the thumbnail will bring
 						up a short description of the project, and clicking it will go to that page -->
-					<div class=portfolioFlex>
-						<!-- This div is the actual portfolio item, and can be replicated if desired -->
-						<div class=projContainer>
-							<img src="https://via.placeholder.com/240x352" alt="Placeholder Image">
+				<div class=portfolioFlex>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html [hunk 4]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
 			<!-- "Portfolio" section will contain links to other projects I have completed -->
-			<div class="linkDummy" id="portfolio"> </div>
-			<section id="portfolioContent">
+			<section id="portfolio">
 				<h1>My Work</h1>
 				<!-- 	This section will be a flexbox containing thumbnails of completed projects. Mousing over the thumbnail will bring
 						up a short description of the project, and clicking it will go to that page -->
-					<div class=portfolioFlex>
-						<!-- This div is the actual portfolio item, and can be replicated if desired -->
-						<div class=projContainer>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html [hunk 5]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
 			</section>
 		
 			<!-- "Portfolio" section will contain links to other projects I have completed -->
-			<div class="linkDummy" id="portfolio"> </div>
-			<section id="portfolioContent">
+			<section id="portfolio">
 				<h1>My Work</h1>
 				<!-- 	This section will be a flexbox containing thumbnails of completed projects. Mousing over the thumbnail will bring
 						up a short description of the project, and clicking it will go to that page -->
-					<div class=portfolioFlex>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html [hunk 6]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
+									<p>HAVE A LOOK</p>
+								</div>
+							</a>
 						</div>
-						
-						<div class=projContainer>
-							<img src="https://via.placeholder.com/240x352" alt="Placeholder Image">
+					</div>
+
+					<div class=projContainer>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html [hunk 7]
```diff
@@ -71,62 +60,76 @@ <h1>About Me</h1>
...
+						<img src="https://via.placeholder.com/240x352" alt="Placeholder Image">
+						<div class=hoverContainer>
```
Comment (ashleycoles) [inline index.html]: Missing quotes

File: index.html (hunk 1 above)
Comment (HiredMussel) [reply index.html]: Oops. Missing quotes should be fixed now for all these cases

Comment (HiredMussel) [reply index.html]: Actually found a few more, which I've also amended

**Pull request merged**